src/routes/classes/[classId]/tests/[attemptId]/+page.svelte

```
@@ -570,27 +576,31 @@
 		isSubmitting = true;
 		try {
 			await flushSync(true);
-				await client.mutation(api.customQuiz.submitAttempt, {
+			await client.mutation(api.customQuiz.submitAttempt, {
 				attemptId,
 				elapsedMs: elapsedMsLocal
 			});
 			if (typeof window !== 'undefined') {
 				window.localStorage.removeItem(cacheKey(String(attemptId)));
 			}
 			await goto(`/classes/${classId}/tests/${attemptId}/results${auto ? '?autoSubmit=1' : ''}`);
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)